core/ide/che-core-ide-generators/src/main/java/org/eclipse/che/util/IDEInjectorGenerator.java

```diff
@@ -113,10 +102,14 @@ public static void generateExtensionManager(File rootFolder) throws IOException
         builder.append("{\n");
         builder.append("\n");
         // define method
-        builder.append(GeneratorUtils.TAB + "/**\n");
-        builder.append(GeneratorUtils.TAB + " * @return the instance of BootstrapController\n");
-        builder.append(GeneratorUtils.TAB + " */\n");
-        builder.append(GeneratorUtils.TAB + "BootstrapController getBootstrapController();\n");
+        builder.append(GeneratorUtils.TAB);
```

Comment:
I think it is better to make it readable by usage of multiple `append` in a line, e.g.
```java
builder.append(GeneratorUtils.TAB).append("/**\n");
builder.append(GeneratorUtils.TAB).append(" * @return the instance of 
BootstrapController\n");
 builder.append(GeneratorUtils.TAB).append(" */\n");
```